src/tests/ftest/util/command_utils.py

```diff
@@ -571,17 +593,24 @@ def _get_result(self, sub_command_list=None, **kwargs):
             getattr(this_command, name).value = value
 
         # Issue the command and store the command result
-        return self.run()
+        return self.run(raise_exception)
 
-    def _get_json_result(self, sub_command_list=None, json_err=False, **kwargs):
+    def _get_json_result(self, sub_command_list=None, json_err=False,
+                         raise_exception=None, **kwargs):
         """Wrap the base _get_result method to force JSON output.
 
         Args:
             sub_command_list (list): List of subcommands.
             json_err (bool, optional): If True, return JSON
                 even if the command fails.
+            raise_exception (bool, optional): whether or not to raise an exception if the command
+                fails. This overrides the self.exit_status_exception
+                setting if defined. Defaults to None.
             kwargs (dict): Parameters for the command.
         """
+
+        self.log.info("CommandWithSubCommand._get_json_result: kwargs=%s", kwargs)
+
```

Comment:
I realize this is still a draft, but presumably this is debug code and will be removed, right?